warehouse/templates/email/unyanked-project-release/body.html

```diff
@@ -0,0 +1,35 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "email/_base/body.html" %}
+
+{% block content %}
+<p>
+  <ul class="removed-project-release">
+    <li>The {{ project }} release {{ release }} released on {{ release_date }} has been yanked.</li>
```

Comment:
```suggestion
    <li>The {{ project }} release {{ release }} released on {{ release_date }} has been un-yanked.</li>
```

Because the email is for "unyanked-project-release".